projects/kit/directives/lazy-loading/lazy-loading.directive.ts

```diff
@@ -18,21 +33,35 @@ export class TuiLazyLoadingDirective {
         this.src$.next(src);
     }
 
+    animation = 'tuiSkeletonVibe ease-in-out 1s infinite alternate';
```

Comment:
I think it would be more obvious to use explicit HostBinding here since now this thing changes

Reply:
Done